### PR TITLE
Prf 7071 remove dead code

### PR DESCRIPTION
**Pull request by ParnikaGupta9**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-7071 Remove the depreciated functionalities- Level four and three](https://app.clickup.com/t/9002019994/PRF-7071)

File: src/card_operations/card_unlock_wallet.c [hunk 1]
```diff
@@ -1,15 +1,16 @@
 /**
- * @file    wallet_locked_controller.c
+ * @file    card_read_verify_shares.c
```
Comment (ujjwal-cyph) [inline src/card_operations/card_unlock_wallet.c:2]: match with the real file name.

File: src/controller_main.c [hunk 2]
```diff
@@ -141,6 +140,11 @@ char arbitrary_data[4096 / 8 + 1];
 
 // TODO: Variable required for code compilation: Delete after all coins are
 // ported
+typedef struct Send_Transaction_Data {
+  uint8_t transaction_confirmation_list_index;
+  txn_metadata transaction_metadata;
+
+} Send_Transaction_Data;
 Send_Transaction_Data var_send_transaction_data = {0};
 Receive_Transaction_Data receive_transaction_data = {0};
```
Comment (ujjwal-cyph) [inline src/controller_main.c:149]: remove these as well.

File: src/card_operations/card_unlock_wallet.c [hunk 3]
```diff
@@ -81,26 +79,6 @@
...
- *****************************************************************************/
-
-/*****************************************************************************
- * STATIC FUNCTIONS
- *****************************************************************************/
-
-/*****************************************************************************
- * GLOBAL FUNCTIONS
- *****************************************************************************/
-
```
Comment (ujjwal-cyph) [inline src/card_operations/card_unlock_wallet.c:103]: undo this for maintaining the template

File: src/card_operations/card_unlock_wallet.h [hunk 4]
```diff
@@ -16,6 +16,8 @@
 #include "flash_struct.h"
 #include "stdbool.h"
 #include "stdint.h"
+#include "ui_instruction.h"
+#include "nfc.h"
```
Comment (ujjwal-cyph) [inline src/card_operations/card_unlock_wallet.h:20]: if possible move these into the source file.

**Review by ujjwal-cyph — CHANGES_REQUESTED** [review]
rest looks okay.

**Pull request closed without merging**